calculatrice.php

```diff
@@ -6,7 +6,7 @@
  */
 function somme($x , $y) 
 {
-    $calcul = $x - $y;
+    $calcul = $x + $y;
```

Comment:
Ca me semble la bonne solution. 
Qu'en pense l'équipe ?
```suggestion
    $calcul = $x + $y;
```